__tests__/__fixtures__/one-module-one-beforeeach-no-hooks-new-import-input-test.js

```diff
@@ -0,0 +1,11 @@
+import { module, test } from 'qunit';
```

Comment:
This test is called `one-module-one-beforeeach-no-hooks-new-import-input-test.js`, but it doesn't appear to have one beforeEach. Was that implying the begin state, or the end state?

Reply:
Ah, copy typo. The file names should imply the begin state.